Travel Scheduler/Views/SlideMenuUIView.h

```diff
@@ -0,0 +1,26 @@
+//
+//  SlideMenuUIView.h
```

Comment:
This is related to a separate feature. Please don't have spillover code across pull requests!

Reply:
Sure....Taken notice of this